net-wireless/wepattack/wepattack-0.1.3-r5.ebuild

```diff
@@ -22,14 +23,21 @@ DEPEND="
 "
 
 RDEPEND="${DEPEND}
-john? ( || ( app-crypt/johntheripper app-crypt/johntheripper-jumbo ) )"
+	john? ( || (
+		app-crypt/johntheripper
+		app-crypt/johntheripper-jumbo
+	)
+)"
 
-S="${WORKDIR}/${MY_P}"
+PATCHES=(
+	"${FILESDIR}"/${P}-filter-mac-address.patch
+	"${FILESDIR}"/${P}-missed-string.h-warnings-fix.patch
+)
 
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-filter-mac-address.patch
-	epatch "${FILESDIR}"/${P}-missed-string.h-warnings-fix.patch
-	chmod +x src/wlan
+	default
+
+	fperms +x src/wlan
```

Comment:
...does this work? `fperms` implicitly adds `${D}`

Reply:
oh you are right, we can't use it in `src_prepare`